src/test/java/com/napier/sem/DAO_IntegrationTests.java

```diff
@@ -75,6 +75,21 @@ void allCountriesIn_arrayIsEmptyWhenAreaNameIsInvalid()
         assertEquals(0, countries.size());
     }
 
+    // Tests that the results are listed in descending order.
+    @Test
+    void allCountriesIn_countriesPopulationIsInDescendingOrder()
+    {
+        // when
+        ArrayList<Record> countries = dao.allCountriesIn(App.CONTINENT, "Asia");
+
+        // then
+        int i = countries.size();
+        for (int x = 0; x < i-1; x++)
```

Comment:
i imagine this will go for the other for loops, i don't think i explained myself well enough so i'll give an example here. we can remove some complexity by doing the following:
```
for (int i =0; i < (countries.size() - 1); i++)
{
    assertTrue(countries.get(i).getPopulation() >= countries.get(i+1).getPopulation());
}
```